routes/userRoutes.js

```diff
@@ -0,0 +1,51 @@
+require("dotenv").config
```

Comment:
Please do not load the env files again, as you are already loading them in the `index.js` file.